extensions/amp-story-subscriptions/validator-amp-story-subscriptions.protoascii

```diff
@@ -15,4 +15,12 @@ tags: {  # <amp-story-subscriptions>
   amp_layout: {
     supported_layouts: CONTAINER
   }
+  attrs: {
+    name: "price"
+    mandatory: true
+  }
+  attrs: {
+    name: "description"
+    mandatory: true
+  }
```

Comment:
You'll need to add the other supported fields as listed in your documentation, otherwise people can't use them: https://github.com/ampproject/amphtml/pull/38179

Reply:
Done. Thanks for the catch.